sdcm/sct_events/group_common_events.py

```diff
@@ -140,21 +140,25 @@ def ignore_disk_quota_exceeded_errors(node):
             db_event=DatabaseLogEvent.BACKTRACE,
             line="Disk quota exceeded",
             node=node,
+            extra_time_to_expiration=3600,  # one hour
```

Comment:
why so long?

Reply:
Because we have log reading lags up to 1 hour?

Reply:
Yes, but even that wouldn't be enough

There are lags bigger than, it's just a temp patch, that might help on short longevity